src/components/DepositWidget/index.tsx

```diff
@@ -151,6 +151,10 @@ const DepositWidget: React.FC = () => {
   }
 
   function _updateToken(tokenAddress: string, updateBalances: Mutation<TokenBalanceDetails>): void {
+    if (!mounted.current) {
```

Comment:
If there's a mounted check inside `_updateToken`, then there's absolutely no need for any mounted checks outside, like L182 and some others

Reply:
True! changed